aws-distributed-configuration/build.gradle.kts

```diff
@@ -4,6 +4,6 @@ plugins {
 
 dependencies {
     api(project(":aws-common"))
-    api("io.micronaut.discovery:micronaut-discovery-client")
+    api(mn.micronaut.discovery)
```

Comment:
I don't understand. 

should not be `artifactId` replacing `-` with `.` and preffixing with `mn.` ?

Reply:
@melix @timyates  ?

Reply:
It only depends on how it was written in the catalog in the beginning

Reply:
And it's been "discovery" for over a year

https://github.com/micronaut-projects/micronaut-core/blob/3.6.x/gradle/libs.versions.toml#L282